org.opent2t.onboarding.insteon/js/thingOnboarding.js

```diff
@@ -0,0 +1,58 @@
+/* jshint esversion: 6 */
+/* jshint node: true */
```

Comment:
Removed.

---
In reply to: [82907678](https://github.com/openT2T/onboarding/pull/21#discussion_r82907678) [](ancestors = 82907678)